recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/gles-user-module_1.7.bb

```diff
@@ -1,26 +1,48 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+require gles-user-module.inc
 
-COMPILER_URL = "${TOPDIR}/../proprietary/rcar/meta/META_Embedded_Toolkit_2.8.1.0.3.1.zip"
+DESCRIPTION = "PowerVR GPU user module"
+LICENSE = "CLOSED"
 
-# we already provide the patched rc.pvr file
-SRC_URI_remove = " \
-    file://change-shell.patch \
+DEPENDS = " \
+    gles-module-egl-headers \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'libgbm wayland-kms', '', d)} \
+    virtual/kernel \
+    llvmpvr \
+    wayland-kms \
 "
+
+PREFERRED_VERSION_gles-module-egl-headers = "1.7"
```

Comment:
this should not be configured in the recipe, IMO
it can be configured in the layer, local.conf (I would prefer the later)